### PR TITLE
Stopwords and sorting language

### PR DESCRIPTION
**Pull request by paulauus**
Added more stopwords so wordcloud looks more informative.
Changed difficult language in sorting to easy language.

### CODE REVIEW

File: dashboard/functions.py [hunk 1]
```diff
@@ -232,20 +232,22 @@ def create_genre_bar_chart(show_nsfw: bool, os_selection: list[str], start_date:
...
 
-
 def preprocess_descriptions(descriptions: list[str]) -> dict[str, int]:
     """
-    Preprocess game descriptions: lowercasing, removing punctuation, filtering stopwords.
+    Preprocess game descriptions: lowercasing, removing punctuation, filtering stopwords,
+    and excluding specific words.
     """
     stop_words = set(nltk_stopwords.words('english'))
+    custom_excluded_words = {"game", "play","new", "must", "get", "one",
```
Comment (Leon-JS02) [inline dashboard/functions.py:241]: This is an interesting list of extra stopwords. We can always add to it later if there appear to be more redundant words in the cloud.

**Review by Leon-JS02 — DISMISSED** [review]
Good words.

File: dashboard/pages/Games.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     )
 with cols[2]:
     today = datetime.now().date()
-    min_date = datetime(2024, 10, 7).date()
+    min_date = datetime(2024, 9, 30).date()
```
Comment (mehrdadhalali) [inline dashboard/pages/Games.py:36]: I LOVE NYMPHOMANIAC: SEX ADDICT

**Review by mehrdadhalali — APPROVED** [review]

**Review by Leon-JS02 — APPROVED** [review]
Reapproval.

File: dashboard/pages/Games.py (hunk 2 above)
Comment (MichaelHassan) [reply dashboard/pages/Games.py:36]: behave

**Pull request merged**